homeassistant/components/tractive/switch.py

```diff
@@ -0,0 +1,167 @@
+"""Support for Tractive switches."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+from aiotractive.exceptions import TractiveError
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import (
+    ATTR_BUZZER,
+    ATTR_LED,
+    ATTR_LIVE_TRACKING,
+    CLIENT,
+    DOMAIN,
+    SERVER_UNAVAILABLE,
+    TRACKABLES,
+    TRACKER_HARDWARE_STATUS_UPDATED,
+)
+from .entity import TractiveEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class TractiveSwitchEntityDescription(SwitchEntityDescription):
+    """Class describing Tractive switch entities."""
+
+    method: str | None = None
```

Comment:
`method` should be a required key (without default) with type
```py
Literal["async_set_buzzer", "async_set_led", "async_set_live_tracking"]
```
just to be safe. You'll need to create a mixin dataclass for that.

Reply:
> You'll need to create a mixin dataclass for that.

I think I need a little help here :)

Reply:
This here is a good example
https://github.com/home-assistant/core/blob/565a9fea6be97c2facb838701992db9df825d3a9/homeassistant/components/meteo_france/const.py#L55-L66

Reply:
Thanks, done.